knowledge-base/inputs-clear-selection-value.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Clear the value (selection) of a combobox, dropdown, input
+description: How to clear the selection of a combo box, dropdown or other input
```

Comment:
```suggestion
description: How to clear the selected value of a combo box, dropdown or other input
```

Reply:
I think I want to keep the original to have more variety on the keywords, also captures the way the question was asked.